src/iotjs_objectwrap.c

```diff
@@ -17,44 +17,30 @@
 #include "iotjs_objectwrap.h"
 
 
-namespace iotjs {
+void iotjs_jobjectwrap_initialize(iotjs_jobjectwrap_t* jobjectwrap,
+                                  const iotjs_jval_t* jobject,
+                                  uintptr_t jhandle,
```

Comment:
`uintptr_t` is introduced in C99. What is our supporting C version? May I assume C99? There are also many use cases using fixed width integer types (e.g. `uint32_t`, ...), which is also introduced in C99, by including `<stdint.h>`. If we decide to support C99 after switching C++ to C, the code `typedef enum { false, true } bool;` in iotjs_def.h could be replaced with including `<stdbool.h>`